src/main/java/Parser.java

```diff
@@ -0,0 +1,71 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+/**
+ * Class for the Parser which parsers commands.
+ */
+public class Parser {
+
+    /**
+     * Parses the command given by the user.
+     *
+     * @param input the string representation of the command input by the user.
+     * @return the command object given by the input.
+     * @throws DukeException thrown when comm
+     */
+    public static Command parse(String input) throws DukeException {
+        if (input.equals("bye")) {
+            return new ExitCommand();
+        } else if (input.length() <= 3) {
+            throw new DukeException("I'm sorry, but I don't know what that means :-(");
+        } else if (input.substring(0, 4).equals("find")) {
```

Comment:
Is there a better method to check whether a string "contains" something? Also does this catch the errors resulting from invalid inputs for find, done, delete, etc?